plugin/pkg/auth/authorizer/node/graph_populator.go

```diff
@@ -157,6 +157,7 @@ func (g *graphPopulator) updatePod(oldObj, obj interface{}) {
 	}
 	klog.V(4).Infof("updatePod %s/%s for node %s", pod.Namespace, pod.Name, pod.Spec.NodeName)
 	g.graph.AddPod(pod)
+	klog.V(4).Infof("pod %s in namespace %s is updated for the node %s", pod.Name, pod.Namespace, pod.Spec.NodeName)
```

Comment:
If the goal is to measure the latency, should we grab an actual timestamp for a more accurate measurement?

I'm also wondering whether this should be `V(5)` (trace level verbosity, according to https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/logging.md)

Reply:
Hi, @tallclair Thanks for taking look at this.
> If the goal is to measure the latency, should we grab an actual timestamp for a more accurate measurement?

Seems good to me.  I will add the timestamp in the logs statement when the update is started and when the update is completed. and same for deletion also. Is it seem good? or Do you have any other suggestions?

> I'm also wondering whether this should be `V(5)` (trace level verbosity, according to https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/logging.md)

And For verbosity you are right, it should be `V(5)`. I will update this also.

Reply:
Hi @tallclair 
I have addressed the comment, PTAL!

Thanks